### PR TITLE
Core component footer

### PR DESCRIPTION
**Pull request by prateek1497**
Major changes in:
- Model
- HTL file
- Dailog file

### CODE REVIEW

File: bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Footer.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ConsumerType;
+
+/**
+ * Defines the form {@code FileInput} Sling Model used for the {@code /apps/core/fd/components/form/fileinput/v1/fileinput} component.
```
Comment (gahuja1991) [inline bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Footer.java:21]: The documentation needs to be updated according to the component

File: it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
                       jcr:primaryType="nt:unstructured"
                       jcr:title="formcontainer-policy"
                       sling:resourceType="wcm/core/components/policy/policy"
-                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker]">
+                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker,/libs/core/fd/components/form/footer/v1/footer]">
```
Comment (gahuja1991) [inline it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml:15]: Lets create a proxy component in apps and use the link there.

File: ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html [hunk 3]
```diff
@@ -37,7 +37,7 @@
               readonly="${text.readOnly}"
               required="${text.required}"
               placeholder="${text.placeHolder}"
-              data-cmp-required-message="${text.constraintMessages.required}">${text.default}</textarea>
+                  data-cmp-required-message="${text.constraintMessages.required}">${text.default}</textarea>
```
Comment (gahuja1991) [inline ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html:40]: I think this got in by mistake.Lets revert this change.

File: bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Footer.java (hunk 1 above)
Comment (prateek1497) [reply bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Footer.java]: right, updated

File: ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html (hunk 3 above)
Comment (prateek1497) [reply ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html]: yeah right, this is done

File: it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml (hunk 2 above)
Comment (prateek1497) [reply it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml]: this is done, in current push, made a component in apps
